src/SlmLocale/Locale/Detector.php

```diff
@@ -191,6 +191,9 @@ public function detect(RequestInterface $request, ResponseInterface $response =
             $events->triggerEventUntil(function ($r) use (&$return) {
                 if ($r instanceof ResponseInterface) {
                     $return = true;
+                } elseif ($r === false) {
+                    // return true, if a previous listener returned false. So we can stop processing further listeners. This is needed for AssetStrategy to prevent redirects.
+                    return true;
```

Comment:
would be nice to have this tested also

Reply:
I do not know exactly how to test this specific part. Maybe someone can help me here?